website/static/js/projectSettings.js

```diff
@@ -8,13 +8,19 @@ var $osf = require('js/osfHelpers');
 var oop = require('js/oop');
 var ChangeMessageMixin = require('js/changeMessage');
 
-var NodeCategorySettings = oop.extend(
+var NodeCategoryTitleDescriptionSettings = oop.extend(
```

Comment:
This is a bit verbose; by convention, we name our constructors by the name of the module, so I think `ProjectSettings` would make sense.